feature/account/setup/src/test/kotlin/app/k9mail/feature/account/setup/ui/options/AccountOptionsStateMapperKtTest.kt

```diff
@@ -0,0 +1,37 @@
+package app.k9mail.feature.account.setup.ui.options
+
+import app.k9mail.feature.account.setup.domain.entity.AccountOptions
+import app.k9mail.feature.account.setup.domain.entity.EmailCheckFrequency
+import app.k9mail.feature.account.setup.domain.entity.EmailDisplayCount
+import app.k9mail.feature.account.setup.domain.input.StringInputField
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import org.junit.Test
+
+class AccountOptionsStateMapperKtTest {
+
+    @Test
+    fun `should map state to account options`() {
+        val state = AccountOptionsContract.State(
+            accountName = StringInputField("accountName"),
+            displayName = StringInputField("displayName"),
+            emailSignature = StringInputField("emailSignature"),
+            checkFrequency = EmailCheckFrequency.Companion.DEFAULT,
+            messageDisplayCount = EmailDisplayCount.DEFAULT,
```

Comment:
Using the default values here implicitly checks the implementation of `EmailCheckFrequency` and `EmailDisplayCount`. But you shouldn't have to touch this test when changing the defaults in either of those classes.